real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/IMUTemperatureMessage.java

```diff
@@ -0,0 +1,61 @@
+package com.roboclub.robobuggy.messages;
```

Comment:
yes, the entire class please. We don't parse temperature anymore (it was an artifact of format 0)